### DISCUSSION TITLE
Is it possible to use chrono NaiveDateTime in struct with welds trait

### DISCUSSION THREAD

**Discussion by waywardcoder** [weldsorm/welds#23] [General]
I am trying to use chrono NaiveDateTime in a struct that has the welds trait, i am getting:

error[E0277]: the trait bound `NaiveDateTime: row::mssql_row_wrapper::TiberiusDecode` is not satisfied
   --> src/repository/models/audit_ledger.rs:6:17
    |
6   | #[derive(Debug, WeldsModel, PartialEq, Default,Serialize,Deserialize)]
    |                 ^^^^^^^^^^ the trait `row::mssql_row_wrapper::TiberiusDecode` is not implemented for `NaiveDateTime`

My cargo.toml:

chrono = { version = "0.4.38",features = ["serde"]}
chrono-tz = "0.9.0"
bb8 = { version="0.8" }
bb8-tiberius = { version="0.15"}
tiberius = { version = "0.12.2", features = ["integrated-auth-gssapi","chrono","tds73","rust_decimal","tokio"] }
tokio = { version = "1.37.0", features = ["macros", "rt-multi-thread"] }
welds = { version = "0.3.1", features = ["mssql"] }
serde = { version = "1.0.197", features = ["derive"] }

**Comment by lex148** [comment]
hey waywardcoder,

So the mssql driver works a bit differently then the sqlx one. I definitely need to work in improving the documentation here. 
Welds can't just use the raw features in tiberius like it can in sqlx. You will need the following in your `cargo.toml` file.

```
welds-connections = { version = "0.3.5", features = ["mssql", "mssql-chrono"] }
```
